runtime/moonriver/tests/integration_test.rs

```diff
@@ -1835,7 +1775,7 @@ fn call_pallet_xcm_with_fee_after_ahm() {
 			let asset = currency_to_asset(CurrencyId::ForeignAsset(asset_id), 100_000_000_000_000);
 			let asset_fee = currency_to_asset(CurrencyId::ForeignAsset(asset_id), 100);
 
-			// Once the AH migration starts, we should no longer be able to use the parent location as reserve.
+			// After the asset hub migration, we can no longer use the parent location as reserve.
```

Comment:
We need to change this test to perform a successful `assert_ok!`, for this we need to use `transfer_assets_using_type_and_then`.